bedrock/firefox/templates/firefox/enterprise/index.html

```diff
@@ -75,95 +77,95 @@ <h2 class="enterprise-section-title">{{ _('Unmatched data protection — on the
     <section id="download" class="enterprise-section enterprise-download">
       <h2 class="enterprise-section-title">{{ _('Enterprise downloads') }}</h2>
 
-      <ul class="mzp-l-card-third enterprise-download-lists">
-        {% call picto_card(
-          title=_('Windows 64-bit'),
-          class='enteprise-download-package win64',
-          heading_level=3,
-          custom_desc=True
-        ) %}
+      <div class="enterprise-download-lists">
+
+        <section class="enterprise-download-block">
+          <div class="enterprise-download-header">
+            <h3 class="enteprise-download-title platform-win64">{{ _('Windows 64-bit') }}</h3>
+
+            <div class="mzp-c-menu-list mzp-t-cta mzp-t-download" id="win64-download-list">
+              <h4 class="mzp-c-menu-list-title">{{ _('Select your download') }}</h4>
+              <ul class="mzp-c-menu-list-list">
+                <li class="mzp-c-menu-list-item"><a href="https://download.mozilla.org/?product=firefox-latest-ssl&os=win64">{{ _('Firefox browser') }}</a></li>
```

Comment:
Side thought - do we need to specify a language for these links, or will download.mozilla.org serve the most appropriate version depending on someone's accept-language header?

Reply:
I'm not sure. They didn't have a language param originally, but I don't know if that means the download defaults to English, or it downloads some all-languages pack, or if bouncer detects language... I suppose I should download one and see what it is. But I could also append the current page language to each URL, and we do have the link to /all below.

Reply:
We also don't need to figure this out straight away, as the page isn't currently localized: https://www.mozilla.org/en-US/firefox/enterprise/